package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -503,6 +538,11 @@ def __init__(self, filename, lammps_coordinate_convention="auto",
                              " is not a valid option. "
                              f"Please choose one of {option_string}")
 
+        if additional_columns:
+            self._additional_columns = additional_columns
```

Comment:
This will likely fail with `additional_columns=True` as input. 

Add error check and test that you catch the bad input.

Reply:
Address this